File: server/router/business/cmsTag.go
package business

import (
	"go-cms/api/v1"
	"go-cms/middleware"
	"github.com/gin-gonic/gin"
)

type CmsTagRouter struct {
}

// InitCmsTagRouter 初始化 CmsTag 路由信息
func (s *CmsTagRouter) InitCmsTagRouter(Router *gin.RouterGroup) {
	router := Router.Group("cmsTag").Use(middleware.OperationRecord())
	routerNoRecord := Router.Group("cmsTag")
	var apiV1 = v1.ApiGroupApp.BusinessApiGroup.CmsTagApi
	{
		router.POST("createCmsTag", apiV1.CreateCmsTag)   // 新建CmsTag
		router.POST("deleteCmsTag", apiV1.DeleteCmsTag) // 删除CmsTag
		router.POST("deleteCmsTagByIds", apiV1.DeleteCmsTagByIds) // 批量删除CmsTag
		router.POST("updateCmsTag", apiV1.UpdateCmsTag)    // 更新CmsTag
	    router.POST("quickEdit", apiV1.QuickEdit)  // 快速编辑	
	}
	{
		routerNoRecord.GET("findCmsTag", apiV1.FindCmsTag)        // 根据ID获取CmsTag
		routerNoRecord.GET("getCmsTagList", apiV1.GetCmsTagList)  // 获取CmsTag列表
        routerNoRecord.GET("excelList", apiV1.ExcelList)  // 分页导出excel CmsTag列表
	}
}

File: web/src/api/cmsTag.ts
import { http,resData } from "@/utils/http";
 
// @Tags CmsTag
// @Summary 创建CmsTag
export const createCmsTag = (data?: object) => {
   return http.request<resData>("post", "/cmsTag/createCmsTag", { data });  
}

// @Tags CmsTag
// @Summary 删除CmsTag
export const deleteCmsTag = (data?: object) => {
   return http.request<resData>("post", "/cmsTag/deleteCmsTag", { data });  
}
 

// @Tags CmsTag
// @Summary 删除ByIdsCmsTag
export const deleteCmsTagByIds = (data?: object) => {
   return http.request<resData>("post", "/cmsTag/deleteCmsTagByIds", { data });  
}
 

// @Tags CmsTag
// @Summary 更新CmsTag
export const updateCmsTag = (data?: object) => {
   return http.request<resData>("post", "/cmsTag/updateCmsTag", { data });  
}

// @Tags CmsTag
// @Summary 快速编辑
export const quickEdit = (data?: object) => {
   return http.request<resData>("post", "/cmsTag/quickEdit", { data });  
}
 

// @Tags CmsTag
// @Summary 用id查询CmsTag
export const findCmsTag = (params?: object) => {
   return http.request<resData>("get", "/cmsTag/findCmsTag", { params });  
}
 

// @Tags CmsTag
// @Summary 分页获取CmsTag列表
export const getCmsTagList = (params?: object) => {
   return http.request<resData>("get", "/cmsTag/getCmsTagList", { params });  
}
 

// @Tags CmsTag
// @Summary 导出excelCmsTag列表
export const excelList = (params?: object) => {
   return http.request<resData>("get", "/cmsTag/excelList", { params });  
}
